setup.py

```diff
@@ -43,6 +42,17 @@
 if not os.getenv("READTHEDOCS"):
     install_requires.append("python-snappy~=0.6.0")
 
+dev_requires = [
```

Comment:
wait so now the environment file has pudl + only dev stuff, so what is this and how does this relate to the environment setup?

Reply:
This is a special case, because `python-snappy` causes problems on RTD (where the underlying binary libsnappy is not installed) and the code isn't actually *running* on RTD, it's just getting imported and parsed. So instead `python-snappy` is "mocked" on RTD, and not installed.  The environment variable `READTHEDOCS` is set to `True` in their build environment, so this is the Python packaging (`setup.py`) trying to figure out whether it's running on RTD or not, and act appropriately.

Reply:
ah i see. leave this as an inline comment to explain this?

Reply:
Oh wait, I thought you were referring to the `if not os.getenv("READTHEDOCS")` thing. Were you talking about `dev_requires` instead? I'm confused about what you're confused about.

I was trying to remove as much duplication of environment specification as possible so the only stuff that's left in `environment.yml` is stuff that shouldn't be installed necessarily when PUDL gets installed, or stuff that specifically benefits from being installed via `conda` -- like the packages with finnicky compiled libraries.